packages/nameguard-react/src/components/TruncatedText/TruncatedText.tsx

```diff
@@ -92,7 +94,7 @@ export const TruncatedText = ({
             }
           >
             {getTextElm(
-              "text-white text-sm leading-5 break-all",
+              `text-white text-sm leading-5 break-all ${tooltipContentStylingClasses}`,
```

Comment:
Should we use cc here?

If so, please update our code everywhere we call `getTextElem`